gustaf/utils/connec.py

```diff
@@ -316,7 +329,11 @@ def make_hexa_volumes(resolutions):
     total_nodes = np.product(nnpd)
     total_volumes = np.product(nnpd - 1)
     node_indices = np.arange(total_nodes, dtype=np.int32).reshape(nnpd[::-1])
-    volumes = np.ones((total_volumes, 8), dtype=np.int32) * int(-1)
+
+    try:
+        volumes = np.ones((total_volumes, 8), dtype=np.int32) * int(-1)
+    except ValueError as e:
+        raise ValueError(f"Problem with generating volumes. {e}")
```

Comment:
```suggestion
    volumes = np.full((total_volumes, 8), -1, dtype=np.int32)
```

Reply:
The problem is, that the total_volumes can be negative and the numpy function throws this exception. I think the best way is to check if the total_numbers is greater than 0.

Reply:
ahh I see, good point! Then maybe we can add this after nnpd = np.asarray(resolutions)
```python
if any(nnpd < 0):
    raise ---
```